app/mailers/user_mailer.rb

```diff
@@ -3,10 +3,14 @@
 class UserMailer < ApplicationMailer
   helper :url, :submissions, :offers
 
+  include UrlHelper
```

Comment:
**thought (non-blocking):** I'm surprised to see that you needed to explicitly include `UrlHelper` because I thought that either (a) helpers were globally accessible, or (b) this helper was included in line 4. Mailers and their templates are analogous to Controllers and their views, so I feel like the "correct" thing to do is call the helper method from the template if it is accessible there.

Reply:
thank you for this tip 🙇‍♂️